app/src/main/res/layout/collections_list_item.xml

```diff
@@ -1,35 +1,65 @@
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+       - License, v. 2.0. If a copy of the MPL was not distributed with this
+       - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/collection_item_tab"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:foreground="?android:attr/selectableItemBackground">
+    android:layout_marginTop="8dp"
+    android:layout_marginBottom="8dp"
+    android:clickable="true"
+    android:clipToPadding="false"
+    android:focusable="true"
+    android:foreground="?android:attr/selectableItemBackground"
+    app:cardBackgroundColor="?above"
+    app:cardCornerRadius="@dimen/tab_corner_radius"
+    app:cardElevation="5dp">
 
-    <TextView
-        android:id="@+id/collection_item"
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_margin="16dp"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="?foundation"
-        android:drawableStart="@drawable/ic_tab_collection"
-        android:drawablePadding="14dp"
-        android:drawableTint="?accent"
-        android:paddingStart="20dp"
-        android:paddingTop="12dp"
-        android:paddingBottom="12dp"
-        android:textColor="?primaryText"
-        android:textSize="16sp"
-        app:layout_constraintBottom_toTopOf="@+id/divider"
-        app:layout_constraintTop_toTopOf="parent"
-        tools:targetApi="m" />
+        android:layout_height="wrap_content">
 
-    <View
-        android:id="@+id/divider"
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
-        android:background="?neutralFaded"
-        app:layout_constraintBottom_toBottomOf="parent" />
-</androidx.constraintlayout.widget.ConstraintLayout>
+        <ImageView
+            android:id="@+id/collection_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_tab_collection"
+            android:tint="@null"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/collection_item"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="12dp"
+            android:ellipsize="end"
+            android:gravity="start"
+            android:maxLines="1"
+            android:minLines="1"
+            android:text="The quick brown fox jumped over the lazy dog"
```

Comment:
We shouldn't have placeholder text in the android:text field because it may flash onscreen while something is loading. That's why there's tools:text and "@tools:sample/lorem". While it probably won't be noticed, placeholders in android:text come with some risk.